raiden-cli/package.json

```diff
@@ -38,9 +39,13 @@
     "webpack-cli": "^3.3.11"
   },
   "dependencies": {
+    "@types/http-errors": "^1.6.3",
     "ethers": "^4.0.47",
     "express": "^4.17.1",
+    "express-validator": "^6.5.0",
```

Comment:
You can validate the inputs, but just would like to note that the SDK is already quite strict about input validation (address, bignumbers, etc), so don't spend too much time on cli's side validation. We can improve errors typing if you need to catch it on client's side, and I'd say this would be better than user validation.

Reply:
Okay. The important point is that we should avoid to validate user data twice. This is just a bad habit and can lead to unexpected behavior. So I agree with you. 

Code this means we need to establish a proper parsing of SDK errors. We would simply forward the user parameter to the SDK and then respond to the user depending on the output/error. This means the SDK must provide us with really good error types and messages. The optimal case would be that we could do a switch case depending on the errors type and the errors message can be forwarded directly. Do you think the SDK provides these requirements already?

Reply:
Just a status update here from me. We decided that the error handling in the SDK for the parameter validation must become better. This is what [that PR](https://github.com/raiden-network/light-client/pull/1649?notification_referrer_id=MDE4Ok5vdGlmaWNhdGlvblRocmVhZDkyNDczNjgxNjoxMjU0MzY0Nw%3D%3D) provides.
Anyways I close this discussion, since there will be a major rework according our decisions. They will require a new review by themselves.

Reply:
Is this dependency still needed? Didn't see it imported anywhere. If not, may you clean it up, please?